cdi/src/main/java/org/hibernate/validator/internal/cdi/ValidatorFactoryBean.java

```diff
@@ -51,7 +53,7 @@
 
 	public ValidatorFactoryBean(BeanManager beanManager, ValidationProviderHelper validationProviderHelper) {
 		this.beanManager = beanManager;
-		this.destructibleResources = newHashSet( 4 );
+		this.destructibleResources = newHashSet( 5 );
```

Comment:
Ah, didn't remember that we had discussed it before. Personally I'd just use the default constructor in the very most cases or pass a size big enough for the few cases where it seems useful.